styles.css

```diff
@@ -0,0 +1,145 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  margin-top: 1.2rem;
+  background-image: url("images/header-bg.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
```

Comment:
- [ ] Good job using your the above image as CSS background image. However, I think it would be better if you move it in your `.headline { ... }` because your app will have some CSS background images on different sections. The current one only concern the `Headline` section and not the entire application